scene/2d/navigation_agent_2d.cpp

```diff
@@ -114,8 +114,53 @@ void NavigationAgent2D::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("link_reached", PropertyInfo(Variant::DICTIONARY, "details")));
 	ADD_SIGNAL(MethodInfo("navigation_finished"));
 	ADD_SIGNAL(MethodInfo("velocity_computed", PropertyInfo(Variant::VECTOR2, "safe_velocity")));
+
+#ifndef DISABLE_DEPRECATED
+	ClassDB::bind_method(D_METHOD("get_final_location"), &NavigationAgent2D::get_final_location);
+	ClassDB::bind_method(D_METHOD("set_target_location", "location"), &NavigationAgent2D::set_target_location);
+	ClassDB::bind_method(D_METHOD("get_target_location"), &NavigationAgent2D::get_target_location);
+	ClassDB::bind_method(D_METHOD("get_next_location"), &NavigationAgent2D::get_next_location);
+#endif // DISABLE_DEPRECATED
+}
+
+#ifndef DISABLE_DEPRECATED
+bool NavigationAgent2D::_set(const StringName &p_name, const Variant &p_value) {
+	if (p_name == "target_location") {
```

Comment:
Target location was never available to be set in the editor, do we need compatibility for it?  Is there any data that will be lost on saved scenes?

Reply:
removed